parsers/SG.py

```diff
@@ -216,12 +219,16 @@ def fetch_production(zone_key='SG', session=None, target_datetime=None, logger=N
         'hydro': 0
     })
 
+    source = 'emcsg.com'
+    if generation_by_type['solar']:
```

Comment:
Nice idea.

Reply:
I'm wondering if it's too clever. When there is no solar production, the data that solar is 0 also somes from EMA...

Reply:
Maybe both should be included by default?